services/ui-src/src/components/modal/Modal.tsx

```diff
@@ -0,0 +1,172 @@
+import React from "react";
+
+// components
+import {
+  Box,
+  Button,
+  Heading,
+  Modal as ChakraModal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from "@chakra-ui/react";
+import { CloseIcon } from "@cmsgov/design-system";
+import { makeMediaQueryClasses } from "utils";
+
+export const Modal = ({ actionFunction, content, modalState }: Props) => {
+  const mqClasses = makeMediaQueryClasses();
+  return (
+    <ChakraModal
+      data-testid="modal"
+      isOpen={modalState.isOpen}
+      onClose={modalState.onClose}
+    >
+      <ModalOverlay />
+      <ModalContent sx={sx.modalContent}>
+        <ModalHeader sx={sx.modalHeader}>
+          <Heading as="h1" sx={sx.modalHeaderText}>
+            {content.heading}
+          </Heading>
+        </ModalHeader>
+        <Box sx={sx.modalCloseContainer}>
+          <Button sx={sx.modalClose} leftIcon={<CloseIcon />} variant="link">
```

Comment:
Close button doesn't do anything right now. Do we need it if we also have cancel? If so, we should add the close function here too

Reply:
Forgot my final merge to push before I went and made some dinner. Good catch 👍